traitsui/ui.py

```diff
@@ -407,8 +407,16 @@ def prepare_ui(self):
 
         # Invoke the handler's 'init' method, and abort if it indicates
         # failure:
-        if handler.init(info) == False:
+        started = handler.init(info)
+        if started is False:
             raise TraitError("User interface creation aborted")
+        elif not isinstance(started, bool):
+            warn(
+                "Handler.init() must return True or False, but instead "
+                f"returned {started}.  "
+                "This will become an error in a future release.",
```

Comment:
This works but it feels off. Without the line length rule, this would be an f-string on a single-line. Not an actionable comment. I think i'd just prefer if it all of them were f-strings. But that might be confusing.

Reply:
Yes, I wasn't sure if it would work like this, or if they all needed to be f-strings, or if an f-string with no substitutions was even allowed.  I tried this and it worked.

If you like I could include explicit `+` operations which might make it less magical.

Reply:
no. it's fine. like i said, not an actionable comment.